### PR TITLE
1password Keys Integration.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:*
https://github.com/bcgov/entity/issues/7849
https://github.com/bcgov/entity/issues/6801

*Description of changes:*
1. change the key new to match new configuration of 1password.
2. change to read the session keys directly instead of read from auth-api-config session key.
3. Update sbc-common-components from 2.4.18 to 2.4.19.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: devops/vaults.json [hunk 1]
```diff
@@ -13,10 +13,39 @@
             "address-key"
         ]
     },
+    {
```
Comment (severinbeauvais) [inline devops/vaults.json:16]: I think I can guess what the purpose of this configuration is and how it works, but is it documented anywhere?

For example, if someone not familiar with this wants to add a "class" of config, or create a new app, or whatever, what do they need to do?

File: src/utils/fetch-config.ts [hunk 2]
```diff
@@ -80,10 +73,14 @@ export async function fetchConfig (): Promise<void> {
...
   addressCompleteKey && console.info('Set Address Complete Key.')
 
-  const ldClientId = response.data['LD_CLIENT_ID']
+  const ldClientId = response.data['BUSINESS_FILING_LD_CLIENT_ID']
   window['ldClientId'] = ldClientId
   ldClientId && console.info('Set Launch Darkly Client ID.')
 
+  const sentryEnable = response.data['SENTRY_ENABLE'];
+  (<any>window).sentryEnable = sentryEnable
+  console.log('Set Sentry DSN Enable.')
```
Comment (severinbeauvais) [inline src/utils/fetch-config.ts:82]: Same as other PR: display value or delete this line.

File: devops/vaults.json (hunk 1 above)
Comment (pwei1018) [reply devops/vaults.json:16]: We have to be careful to add any values to this file. It should be done by Senior developer or SRE team member.

**Review by severinbeauvais — APPROVED** [review]

**Review by andrepestana-aot — APPROVED** [review]
Looks good to me!

**Pull request merged**